setup/upgrade/considerations.md

```diff
@@ -3,11 +3,27 @@ title: "Update considerations"
 description: Learn how to update your Coder deployment.
 ---
 
-The update page provides instructions on how to update your Coder deployment.
+The upgrade page provides instructions on how to upgrade your Coder deployment.
 This article, however, includes information you should be aware of prior to
-updating, such as architecture updates and breaking changes.
+upgrading, such as architecture updates and breaking changes.
 
-## Updating from v1.24 to v1.25
+## Upgrading from v1.25 to v1.26
+
+- Beginning with `1.26`, Coder requires the use of Kubernetes `1.20` or later.
+  See Coder's [version support policy] for more information.
+
+<!-- Turn off linting to avoid changing the link -->
+<!-- markdownlint-disable MD044 -->
+
+[version support policy]: ../kubernetes/index.md#supported-kubernetes-versions
+
+- Coder supports the use of Markdown formatting in system and service banners.
+  Coder now renders the Markdown content in existing banners, instead of
+  displaying the raw Markdown syntax. Furthermore, users who have enabled the
+  beta version via feature flag will need to re-enable this; Coder won't
+  auto-apply the theme.
```

Comment:
I think this mixes up the markdown and the dark theme. Markdown didn't have a public feature flag AFAIK, but dark theme did (and requires manual migration). This should be split into two points